StyleCop.Analyzers/StyleCop.Analyzers/Helpers/SpecializedTasks.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace StyleCop.Analyzers.Helpers
+{
+    using System.Threading.Tasks;
+
+    internal static class SpecializedTasks
+    {
+        internal static Task CompletedTask { get; } = Task.FromResult(default(VoidResult));
```

Comment:
The big downside of this compared to `Task.CompletedTask` is we don't create the task with `InternalTaskOptions.DoNotDispose`. If someone calls `SpecializedTasks.CompletedTask.Dispose()`, this property will become unusable until the process is restarted.